app/js/lib/server/webpush.js

```diff
@@ -31,32 +31,24 @@ export default class WebPush extends EventDispatcher {
       this[p.listenForMessages].bind(this));
 
     return navigator.serviceWorker.ready
-      .then((reg) => {
-        reg.pushManager.getSubscription()
-          .then((existing) => {
-            if (existing && !resubscribe) {
-              // nothing left to do
-              // @todo We should check if we have this subscription on the
-              // server, if not re-register it on calendar's server.
-              // https://github.com/fxbox/calendar/issues/20
-              return;
-            }
+      .then(
+        (reg) => reg.pushManager.getSubscription()
```

Comment:
actually I'm not even sure this is needed, it seems that "subscribe" actually returns an existing subscription if it exists...